src/uu/du/src/du.rs

```diff
@@ -277,8 +282,14 @@ fn du(
 
         for f in read {
             match f {
-                Ok(entry) => match Stat::new(entry.path()) {
+                Ok(entry) => match Stat::new(entry.path(), &options) {
```

Comment:
There's a clippy warning here. You can remove the `&` because `options` is already of type `&Options`